### PR TITLE
cloud: Add updated_urls_topic to cloud function env

### PR DESCRIPTION
**Pull request by octonawish-akcodes**
Added `updated_urls_topic` to the cloud function environment.

Need reviews @spbnick

### CODE REVIEW

File: cloud [hunk 1]
```diff
@@ -1342,6 +1344,7 @@ function cloud_functions_env() {
         [KCIDB_CLEAN_TEST_DATABASES]="$clean_test_databases"
         [KCIDB_EMPTY_TEST_DATABASES]="$empty_test_databases"
         [KCIDB_UPDATED_QUEUE_TOPIC]="$updated_topic"
+        [KCIDB_CACHE_URLS_TOPIC]="$updated_urls_topic"
```
Comment (spbnick) [inline cloud:1347]: The environment variable name needs to follow consistent naming. It should be `KCIDB_UPDATED_URLS_TOPIC`.

**Review by spbnick — CHANGES_REQUESTED** [review]
Looks good so far, with one naming comment inline, but please actually pass the topic name to the function, from `execute_command()`.

File: cloud (hunk 1 above)
Comment (octonawish-akcodes) [reply cloud]: okay

**Comment by spbnick** [comment]
LGTM! Thank you, Abhishek.

**Review by spbnick — APPROVED** [review]

**Comment by spbnick** [comment]
Let's wait for CI to finish.

**Comment by octonawish-akcodes** [comment]
> Let's wait for CI to finish.

Okay

**Pull request merged**